contrib/neml.mk

```diff
@@ -25,7 +25,7 @@ $(NEML_DIR)/src/%.$(obj-suffix) : $(NEML_DIR)/src/%.cxx
 
 ADDITIONAL_INCLUDES  += -I$(neml_includes) $(neml_dep_includes)
 ADDITIONAL_LIBS      += -L$(NEML_DIR) -lneml-$(METHOD)
-ADDITIONAL_CPPFLAGS  += -DNEML_ENABLED
+ADDITIONAL_CPPFLAGS  += -DNEML_ENABLED -DNEML_STRAIN_RATE_LIMIT=1e10
```

Comment:
what does this new flag do?

Reply:
It is needed as a threshold value for inelastic strain rata are considered "too large" (i.e will result in floating point errors)